Sources/PrinceOfVersions/Objective-C Helpers/ResponseModel Wrapers/UpdateData/UpdateResultObjectiveC.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  UpdateResultObjectiveC.swift
+//  PrinceOfVersions
+//
+//  Created by Jasmin Abou Aldan on 13/09/2019.
+//  Copyright © 2019 Infinum Ltd. All rights reserved.
+//
+
+import Foundation
+
+@objcMembers
+public class UpdateResultObject: NSObject {
```

Comment:
Should we hide this for swift?

Reply:
No. Same reason as other ObjC exposed classes.